libraries/designsystem/src/test/kotlin/io/element/android/libraries/designsystem/colors/AvatarColorsTest.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.designsystem.colors
+
+import com.google.common.truth.Truth.assertThat
+import io.element.android.libraries.theme.colors.avatarColorsDark
+import io.element.android.libraries.theme.colors.avatarColorsLight
+import org.junit.Test
+
+class AvatarColorsTest {
+
+    @Test
+    fun `ensure list size`() {
+        // avatarColorsDark and avatarColorsLight size must not be modified.
```

Comment:
I'd rather make the `String.toHash()` function composable and use the underlying `size` property for the modulo, but that works too.

Reply:
I have updated the code, `String.toHash()` does not need to be `@Composable`